code/game/machinery/ARES/ARES_interface_admin.dm

```diff
@@ -231,6 +231,23 @@
 		logged_access += list(current_ticket)
 	data["access_tickets"] = logged_access
 
+	var/list/security_vents = list()
+	for(var/obj/structure/pipes/vents/pump/no_boom/gas/vent in linked_vents)
+		if(!vent.vent_tag)
+			vent.vent_tag = "Security Vent #[tag_num]"
+			tag_num++
+
+		var/list/current_vent = list()
+		var/is_available = TRUE
```

Comment:
Can just assign `is_available` to the result of the COOLDOWN_FINISHED rather than a separate if check